in-viewport.js

```diff
@@ -94,20 +93,35 @@ function createInViewport(container) {
   setInterval(debouncedCheck, 150);
 
   function isInViewport(elt, offset, cb) {
-    var visible = isVisible(elt, offset);
-    if (visible) {
-      if (cb) {
-        watching.splice(indexOf.call(watching, elt), 1);
-        cb(elt);
-      }
-      return true;
+    if (!cb) {
+      return isVisible(elt, offset);
+    }
+
+    var accessor = createAccessor(elt, offset, cb);
```

Comment:
I had not well understood the issue about multiple watch/watchers etc..

Maybe we can call it `remote`. remote.watch() remote.dispose()?